src/test/java/eu/europeana/metis/sandbox/service/util/XsltUrlUpdateServiceImplTest.java

```diff
@@ -30,6 +32,7 @@
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.integration.support.locks.LockRegistry;
 
 @ExtendWith({MockitoExtension.class})
 @TestMethodOrder(OrderAnnotation.class)
```

Comment:
Comment not blocking this current pr:

I just saw that and although it is not related to this pr I wanted to say that I personally try to avoid ordering of unit tests and if I need order I would supply it directly in code.  
Also it seems to me that an order is not really required here since the db repo is in fact mocked.  
Fruit for thought.

Reply:
In a certain moment of the development the order was required because it was using a "local mongo" for this test so it matter the order running the test while changing the db. Maybe writing a big method that sets the order in code instead of using annotations you mean?

Reply:
Yes. Then each unit test is independent.
And also other orders could also be checked, orders for example that should fail.

Reply:
created followup ticket https://europeana.atlassian.net/browse/MET-5084